services/github/src/github-auth.js

```diff
@@ -4,6 +4,31 @@ const { retry } = require("@octokit/plugin-retry");
 
 const PluggedOctokit = Octokit.plugin(retry);
 
+const tokenCache = new Map();
+/**
+ * Github app access token have an expiration date of 1h
+ * https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#authenticating-as-an-installation
+ *
+ * As we run multiple handlers for the same repository, we can save a lot of resources by using cached token
+ * On average it takes 0.5-1s to get one.
```

Comment:
Nice! 🎉

Reply:
Nice catch!  The cache size will be on the order of the number of installations, so in the 10's at most, so it should be pretty small.